EventFilter/TotemRawToDigi/test/emulated_data_test.py

```diff
@@ -0,0 +1,42 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("TotemIntegratedRawDataTest")
+
+# minimum of logs
+process.MessageLogger = cms.Service("MessageLogger",
+    statistics = cms.untracked.vstring(),
+    destinations = cms.untracked.vstring('cerr'),
+    cerr = cms.untracked.PSet(
+        threshold = cms.untracked.string('WARNING')
+    )
+)
+
+# raw data source
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring('file:/afs/cern.ch/user/j/jkaspar/public/run268608_ls0001_streamA_StorageManager.root')
+)
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(10)
+)
+
+# raw-to-digi conversion
+process.load('CondFormats.TotemReadoutObjects.TotemDAQMappingESSourceXML_cfi')
```

Comment:
TotemDAQMappingESSourceXML is badly written: it derives file name location based on CMSSW_BASE.
FileInPath should be used instead
cms.FileInPath
So, this code can not be run from a release distribution